src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -51,7 +51,7 @@ namespace System.Management.Automation
     ///     - There is no ITypeDescriptorContext.
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-linux`
Language Primitive Tests.Test convertion with .Net Core intrinsic type convertor
```powershell
Exception calling "ConvertTo" with "2" argument(s): "Cannot convert the "2,3" value of type "System.String" to type "System.Drawing.Point"."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/LanguagePrimitive.Tests.ps1: line 32
32:         $result = [System.Management.Automation.LanguagePrimitives]::ConvertTo('2,3', [System.Drawing.Point])

```
New-ModuleManifest tests.Relative URIs are not allowed
```powershell
Cannot convert value "../foo" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Module/NewModuleManifest.Tests.ps1: line 115
115:         $testUri = [Uri]"../foo"

```
ConvertTo-Html Tests.Test ConvertTo-Html using page parameters
```powershell
Cannot bind parameter 'CssUri'. Cannot convert value "page.css" to type "System.Uri". Error: "Invalid URI: The format of the URI could not be determined."
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Html.Tests.ps1: line 83
83:         $returnString = ($customObject | ConvertTo-Html -Title "Custom Object" -Body "Body Text" -CssUri "page.css" -As List) -join $newLine

```